src/notfication/notify.py

```diff
@@ -0,0 +1,28 @@
+from email.message import EmailMessage
+import smtplib
+import os
+import ssl
+# from dotenv import load_dotenv
+# load_dotenv()
+
+class Notify:
+    # __serviceAccEmail = os.getenv('SERVICE_EMAIL')
+    # __serviceAccPassword = os.getenv('SERVICE_PASSWORD')
+
+    _subject = " Something went wrong !!"
```

Comment:
Let's the caller handle sending the subject.\
We might want to also add the service name that crashed in the subject.\
Example: "Discord service crashed!"

Reply:
ok